### PR TITLE
Feature/upstream call

### PR DESCRIPTION
**Pull request by hunght**
- handler encode and decode protobuf on upstream class
- implement Message's Key schema in database
- write unit test

### CODE REVIEW

File: __tests__/src/services/upstream/response.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { api_proto as apiProto } from '../../../../src/services/upstream/compiled';
+
+const { Response } = apiProto;
+
+const message = {
+
+  dRKeyStoreGet: {
+    messageKey: 1,
```
Comment (florianlenz) [inline __tests__/src/services/upstream/response.js]: As I already said, those values are invalid. Please checkout the protbuf and test with real values.

File: __tests__/src/services/upstream/upstream.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  });
+});
+
+
+describe('Upstream class', () => {
+  test('Initialize the upstream service', () => {
+    const { Request } = apiProto;
+    const encodedRequest = Request.encode(message).finish();
+    const upstream = new Upstream();
+    upstream.handleRequest(encodedRequest);
```
Comment (florianlenz) [inline __tests__/src/services/upstream/upstream.js]: Where is the assertion .... ?

File: __tests__/src/services/upstream/upstream.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  });
+  // test('handleDRKeyStorePut', async () => {
```
Comment (florianlenz) [inline __tests__/src/services/upstream/upstream.js]: Please remove comments.

File: __tests__/src/services/upstream/upstream.js (hunk 2 above)
Comment (hunght) [reply __tests__/src/services/upstream/upstream.js]: fixed this

**Pull request merged**